ui/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '@splunk/react-ui/Button';
+
+interface DownloadButtonProps {
+    // needs to be same domain if not it will jsut open link
+    fileUrl: string;
+    buttonText: string;
+    fileNameAfterDownload: string;
+}
+
+function DownloadButton(props: DownloadButtonProps) {
+    return (
+        <a
+            target="_blank"
+            href={props.fileUrl}
+            rel="noopener noreferrer"
+            download={props.fileNameAfterDownload}
+            data-test="downloadButton"
```

Comment:
Maybe `downloadOpenAPIButton` or something similar in case we have more similar buttons in the future?

Reply:
yea, currently there are are 2 components, one is general download button, which basically is used to make the simplistic logic, and the second one `OpenApiDownloadButton` is actual usage of that button specifically for openapi.json, i can merge it in one if needed.

Reply:
You are right, by this comment I meant - is a "Download openAPI.json" button going to have some unique selector we can use to write UI test against?